src/app/team/team.module.ts

```diff
@@ -1,19 +1,16 @@
+import { PeopleListModule } from './../shared/people-list/people-list.module';
```

Comment:
Sort import. Also, dot at the beginning of the relative path is not necessary, it is infered:
```suggestion
import { PeopleListModule } from '../shared/people-list/people-list.module';
```